helm-charts/FATE/templates/rollsite-module.yaml

```diff
@@ -24,9 +24,9 @@ data:
             "default": {
                 "default": [
                     {
-                      {{- with .Values.exchange }}
-                        "ip": "{{ .partyIp }}",
-                        "port": {{ .partyPort }}
+                      {{- if .Values.modules.exchange.include }}
+                        "ip": "exchange",
+                        "port": 9390
                       {{- else }}
```

Comment:
The exchange here needs to be configurable, and it is impossible for any exchange to be in the same cluster

Reply:
We can configure different exchange servers in the fedList of exchange module(actually a Nginx server). May be we can change the name of exchange module to avoid the misunderstanding.

Reply:
The new roll site is lbrollsite, so the original rollsite does not need to be modified